test/e2e/testnet/manifest.go

```diff
@@ -0,0 +1,77 @@
+package testnet
+
+import (
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v2/app"
+	"github.com/celestiaorg/celestia-app/v2/app/encoding"
+	"github.com/celestiaorg/celestia-app/v2/test/util/genesis"
+	blobtypes "github.com/celestiaorg/celestia-app/v2/x/blob/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+// Manifest defines the parameters for a testnet.
+type Manifest struct {
+	ChainID      string
+	TestDuration time.Duration
+	// Number of validators in the testnet
+	Validators int
+	TxClients  int
```

Comment:
Can you also document what `TxClients` mean? Could be addressed in the subsequent PR you will be openinig.

Reply:
Of course, please see #3457

Reply:
Well, I ended up addressing it in the current PR, please see https://github.com/celestiaorg/celestia-app/pull/3391/commits/3044f701f1eec4fd93a4ec435475c90d1dfed4fa